interface/forms/group_attendance/functions.php

```diff
@@ -128,7 +128,7 @@ function insert_patient_appt($pid, $gid, $pc_aid, $pc_eventDate, $pc_startTime,
  */
 function insert_patient_encounter($pid, $gid, $group_encounter_date, $participantData, $pc_aid){
     $select_sql = "SELECT id, encounter FROM form_encounter WHERE pid = ? AND external_id = ? AND pc_catid = ? AND date = ?; ";
-    $result = sqlStatement($select_sql, array($pid, $gid, 1000, $group_encounter_date));
+    $result = sqlStatement($select_sql, array($pid, $gid, 15, $group_encounter_date));
```

Comment:
should not hard-code 15 (for now would just insert the pc_catid of the lowest id you get when query 'SELECT pc_catid FROM openemr_postcalendar_categories WHERE pc_cattype = 3 AND pc_active = 1' )